core/templates/components/organisms/accordions/accordions.twig

```diff
@@ -1,4 +1,4 @@
-<ul class="decanter-accordion {{ classes }} {{ modifier_class }}">
+<ul class="decanter-accordion {{ modifier_class }} {{ modifier_class }}">
```

Comment:
This block is missing its attributes twig variable.
Double modifier_class option.